test/sqllogictest/transform/fold_vs_dataflow/1_numbers_dataflow.slt

```diff
@@ -36,7 +36,7 @@ query T multiline
 EXPLAIN SELECT * FROM t_using_dataflow_rendering;
 ----
 Explained Query:
-  Get materialize.public.t_using_dataflow_rendering
+  ReadStorage materialize.public.t_using_dataflow_rendering
```

Comment:
The `Get` -> `ReadStorage` changes are fine (from https://github.com/MaterializeInc/materialize/pull/21571).